symfony/plugins/orangehrmMaintenancePlugin/Controller/PurgeEmployeeController.php

```diff
@@ -33,4 +39,15 @@ public function preRender(Request $request): void
         $component = new Component('purge-employee');
         $this->setComponent($component);
     }
+
+    /**
+     * @inheritDoc
+     */
+    public function handle(Request $request)
```

Comment:
missing return type

Reply:
union types are not supported on PHP 7.4